taxonomies/ingredients_processing.txt

```diff
@@ -2125,48 +2343,48 @@ nl: ongezouten
 pl: bez soli, niesolone
 pt: sem sal, não salgado
 
-en: sweetened
-bg: подсладено, подсладен, подсладена, подсладени
-de: gesüßt, gesüßte
-es: endulzado, endulzada, endulzados, endulzadas
-fi: makeutettu, makeutetut
-hr: zaslađene
-hu: édesített
-it: addolcito, addolciti, addolcita, addolcite
-nl: gezoet, gezoete
-pl: słodzony, słodzona, słodzone, słodzonych, słodzonej, słodzonego
-pt: adoçado, adoçada, adoçados, adoçadas, adocicado, adocicada, adocicados, adocicadas
-sv: sötad
+# sweetened is not a processing as it means an extra ingredient. 
+# It changes an ingredients into a 'compound' ingredient.
+# en: sweetened, sugared, with sugar
```

Comment:
I said this in a previous comment, but sweetening and sugaring are two different (if related) processes. It won’t matter (for this file) if we end up move these out of processing, but, as per my other comment, I think that’s for a different PR and a different discussion. :)